src/components/views/elements/EventListSummary.tsx

```diff
@@ -77,18 +81,37 @@ const EventListSummary: React.FC<IProps> = ({
         </React.Fragment>;
     } else {
         const avatars = summaryMembers.map((m) => <MemberAvatar key={m.userId} member={m} width={14} height={14} />);
-        body = (
-            <div className="mx_EventTile_line">
-                <div className="mx_EventTile_info">
-                    <span className="mx_EventListSummary_avatars" onClick={toggleExpanded}>
-                        { avatars }
-                    </span>
-                    <span className="mx_TextualEvent mx_EventListSummary_summary">
-                        { summaryText }
-                    </span>
+        if (layout == Layout.Bubble) {
```

Comment:
```suggestion
        if (layout === Layout.Bubble) {
```

for consistency

Reply:
Weren't somewhere a change three to two? :thinking: 
So three is what should be used?

Reply:
Generally, both are used all over the codebase, though I think we should use `===` if we can. 

> Weren't somewhere a change three to two?

Do you mean one of my suggestions? If so, I don't thinks

Reply:
Yeah, that's what I thought. Then I've misseen it.